gpu-simulator/trace-driven/trace_driven.cc

```diff
@@ -640,18 +651,10 @@ void trace_shader_core_ctx::func_exec_inst(warp_inst_t &inst) {
       checkExecutionStatusAndUpdate(inst, t, tid);
     }
   }
-
   // here, we generate memory acessess and set the status if thread (done?)
   if (inst.is_load() || inst.is_store()) {
     inst.generate_mem_accesses();
   }
-
-  trace_shd_warp_t *m_trace_warp =
-      static_cast<trace_shd_warp_t *>(m_warp[inst.warp_id()]);
-  if (m_trace_warp->trace_done() && m_trace_warp->functional_done()) {
-    m_trace_warp->ibuffer_flush();
```

Comment:
And now we don't have to flush the iBuffer because we did not fill it with bad values right?